src/components/elementView/elementTable.tsx

```diff
@@ -76,7 +76,7 @@ import "./elementView.scss";
 import FilterSelect from "./filterSelect";
 import { PowerView } from "./powerView/powerView";
 import "./powerView/powerView.scss";
-import { isNullOrUndefined} from "util";
+import { isNullOrUndefined } from "util";
```

Comment:
changes to this file are just linting - I had a made a change in here, linted and saved, then later undid that change, so only linting is left behind